Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -119,6 +119,17 @@ class CartViewModel: BaseViewModel {
         return cart.value?.lineItems?[indexPath.row].totalPrice?.description ?? "N/A"
     }
 
+    func productDetailsViewModelForLineItemAtIndexPath(indexPath: NSIndexPath) -> ProductViewModel {
+        var product = ProductProjection()
+        if let lineItem = cart.value?.lineItems?[indexPath.row] {
+            product.id = lineItem.productId
+            // When navigating to PDP from LineItem, we only show one variant
+            product.masterVariant = lineItem.variant
+            product.name = lineItem.name
```

Comment:
Thank you both a lot for enlightening me 😄 And please pardon my uneducated comment from above 😊 

In that case, there's so reason why not show the full PDP.

I would just preselect the variant from the line item, if you agree?